site-scrapers/Atrius.js

```diff
@@ -1,6 +1,7 @@
 const sites = require('../data/sites.json');
 const https = require('https');
 const html_parser = require('node-html-parser');
```

Comment:
these dependencies are no longer used in this file

Reply:
removed